src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Public.Data/DataSetService.cs

```diff
@@ -52,11 +52,11 @@ internal class DataSetService(
                     .Paginate(page: page, pageSize: pageSize)
                     .ToListAsync(cancellationToken);
 
-                var releasesByDataSetVersionByDataSet =
-                    await GetPreviousReleasesByVersionByDataSet(dataSets, cancellationToken);
+                var releaseVersionsByDataSetVersionIdByDataSetId =
```

Comment:
I see the reasoning behind the rename, but this is quite a hefty variable name and I can't quite get my head around what it represents!  Happy to approve the PR but I think that possibly it might be clearer if the Dictionary of Dictionaries that we return from `GetPreviousReleaseVersionsByVersionIdByDataSetId` was actually some class / record with a meaningful name rather than a nested Dictionary.

One to mull over!

Reply:
It is a bit lengthy, I must admit!

I'll create a private record that I can use for it. Please let me know if you think it's any better :)